pages/linux/torsocks.md

```diff
@@ -0,0 +1,12 @@
+# torsocks
+
+> Use an external application through the Tor network.
+> More information: <https://github.com/dgoulet/torsocks>.
+
+- Run a command using Tor:
+
+`torsocks {{command}}`
+
+- Toggle your shell in/out of Tor mode:
```

Comment:
Hello, thanks for the contribution. I have the following the suggestion, I believe the description is shorter.

```suggestion
- Enable or disable Tor:
```

Reply:
> Hello, thanks for the contribution. I have the following the suggestion, I believe the description is shorter.

I think that this doesn't sufficiently convey that the command only toggles TOR mode for *this shell*. I would be fine with the original version.

Reply:
```suggestion
- Enable or disable Tor in this shell:
```
Considering @gutjuri's cocern, maybe we could go with this?

Reply:
I agree